translations/frontend/en.json

```
@@ -1782,7 +1782,8 @@
                             "show_all": "Show all",
                             "show_disabled": "Show disabled entities",
                             "show_readonly": "Show read-only entities",
-                            "show_unavailable": "Show unavailable entities"
+                            "show_unavailable": "Show unavailable entities",
+							"show_with_area": "Show entities with assigned area"
```

We use spaces, not tabs
```suggestion
                    "show_with_area": "Show entities with assigned area"
```